### ISSUE TITLE
60 FPS Video Support

### ISSUE THREAD

**Issue by IsaacMarovitz** [bluesky-social/social-app#5347]
**Is your feature request related to a problem? Please describe.**

The quality of videos on Bluesky compared to other platforms like Twitter or Mastodon could be better. I've uploaded the same gameplay clip to Bluesky and Twitter and put them side by side and you can see the clear difference:

https://github.com/user-attachments/assets/d49d3efc-1e5d-41d4-8436-1f2cee999925

Digging into the metadata of the videos reveals that the bitrate is fairly similar, but the Bluesky video is restricted to 30 FPS:

Twitter
<img width="1306" alt="twitter" src="https://github.com/user-attachments/assets/ba865bf8-1b4d-4f2f-8179-4510d3944e51">

Bluesky
<img width="1096" alt="mastodon" src="https://github.com/user-attachments/assets/e76b9031-e6d8-46da-bda0-83e84a4913f1">

**Describe the solution you'd like**

A simple solution to this degraded quality would be to allow 60 FPS videos on Bluesky and perhaps a slightly higher bitrate, which would clearly cost storage space.

Twitter: 9.9 MB
Bluesky: 6.7 MB

**Describe alternatives you've considered**

Aside from better encoding formats, little can be done to improve the quality of videos without bumping up the bitrate and/or frame rate.

**Comment by JohnCharlie2012** [comment]
Oh this is a very interesting discovery because BlueSky doesn't specify these limitations on their website. Out of curiosity, what did you use to identify the metadata for these videos? I'd like to utilize that and examine it more with my videos.

**Comment by Invertex** [comment]
> Oh this is a very interesting discovery because BlueSky doesn't specify these limitations on their website. Out of curiosity, what did you use to identify the metadata for these videos? I'd like to utilize that and examine it more with my videos.

Use a method to download the file, whether that's by inspecting the Network tab in your browser's Developer console, or using an extension/app.

Then you can use MediaInfo to view info about the files. https://mediaarea.net/en/MediaInfo

**Comment by IsaacMarovitz** [comment]
> Out of curiosity, what did you use to identify the metadata for these videos? I'd like to utilize that and examine it more with my videos.

I used ffmpeg

**Comment by jeremyfa** [comment]
+1 for adding 60FPS support. On some video game captures, 30FPS can feel very jittery. Problem doesn't exist on Mastodon or Twitter/X

**Comment by jakearchibald** [comment]
I'd also like this for posting screen caps, eg https://bsky.app/profile/jakearchibald.com/post/3m3mzmq7lrs2t

**Comment by FreyaHolmer** [comment]
any update on this? I'd really like support for 60 fps, without it, gameplay footage and animations on bsky feel very choppy and just, kinda outdated

**Comment by WangleLine** [comment]
rly desperate for 60fps videos, as a gamedev posting gameplay clips to bsky

**Comment by jakearchibald** [comment]
I've been posting [videos about Firefox releases](https://bsky.app/profile/webdevs.firefox.com/post/3mhxtjn6uws2c) and they look so much better on X & Mastodon because they do 60fps.

**Comment by seascape** [comment]
Agree. I just posted on the [other 60fps issue](https://github.com/bluesky-social/social-app/issues/8356) about how I wanted to start posting gameplay clips here but the low bitrate and especially 30fps just ruins the quality. Really discouraging.
